src/LaunchDarkly/GuzzleFeatureRequester.php

```diff
@@ -41,10 +39,13 @@ function __construct($baseUri, $apiKey, $options) {
      * @param $key string feature key
      * @return array|null The decoded JSON feature data, or null if missing
      */
-    public function get($key) {
+    public function get($key)
+    {
         try {
-            $response = $this->_client->get("/api/eval/features/$key");
-            return $response->json();
+            $uri = $this->_baseUri . "/api/eval/features/$key";
```

Comment:
There are some behavior changes in the Guzzle 6.2.1 client- hence all these non-caching related changes. One of them is discussed here: https://github.com/guzzle/guzzle/issues/1532